src/components/Header/Header.js

```diff
@@ -1,9 +1,100 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/role-supports-aria-props */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from 'react';
+//import { Link } from 'react-router-dom';
 
 function Header() {
+  const [showDropDown, setShow] = useState(false);
+
   return (
     <header>
-      <h1>Our React Starter</h1>
+      <nav className="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
+        <a className="navbar-brand" href="#">
+          Navbar
+        </a>
+        <button
+          className="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarsExampleDefault"
+          aria-controls="navbarsExampleDefault"
+          aria-label="Toggle navigation"
+          aria-expanded="false"
+        >
+          <span className="navbar-toggler-icon"></span>
+        </button>
+        <div className="collapse navbar-collapse" id="navbarsExampleDefault">
+          <ul className="navbar-nav mr-auto">
+            <li className="nav-item active">
+              <a className="nav-link" href="/">
+                Home
+                <span className="sr-only">(current)</span>
+              </a>
+            </li>
+            <li className="nav-item">
+              <a className="nav-link" href="/shoppingcart">
+                Shopping Cart
+              </a>
+            </li>
+            {/* <li className="nav-item">
+              <a
+                className="nav-link"
+                href="#"
+                tabindex="-1"
+                aria-disabled="true"
+              >
+                Disabled
+              </a>
+            </li> */}
+            <li className="nav-item dropdown">
+              <a
+                className="nav-link dropdown-toggle"
+                href="#"
+                id="dropdown01"
+                data-toggle="dropdown"
+                aria-haspopup="true"
+                //   using a hook to control dropdown behaviour instead of bootstraps js
+                aria-expanded={showDropDown === true ? 'true' : 'false'}
```

Comment:
this this <a> tag is already using data-toggle, you can set data-toggle="showDropDown" and keep aria-expanded="false". Either that or get rid of the data-toggle.